### PR TITLE
Gtfs docs

### PR DESCRIPTION
**Pull request by edasmalchi**
# Description
Docs page with a link to @lauriemerrell video and a few notes for our new analysts! May expand in the future.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

## How has this been tested?
_Include commands/logs/screenshots as relevant._

## Post-merge follow-ups
_Document any actions that must be taken post-merge to deploy or otherwise implement the changes in this PR (for example, running a full refresh of some incremental model in dbt). If these actions will take more than a few hours after the merge or if they will be completed by someone other than the PR author, please create a dedicated follow-up issue and link it here to track resolution._

- [ ] No action required
- [ ] Actions required (specified below)

### CODE REVIEW

File: docs/warehouse/what_is_gtfs.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+(what-is-gtfs)=
+# What is GTFS, anyway
```
Comment (atvaccaro) [inline docs/warehouse/what_is_gtfs.md:2]: nit: ending question mark

File: docs/warehouse/what_is_gtfs.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Video Introduction
+
+### [Start by watching this amazing video that Laurie made!](https://www.youtube.com/watch?v=Us6j5GYoLtk)
+
+### Video Notes
+
+* Laurie mentions downloading an example feed to look at the files. This is a great idea! But when working with our data warehouse remember that you don't have to interact with raw GTFS feeds directly (lots of important work has been done for you!). Still, we reccomend taking a look at an example feed to understand what it looks like. Here's one from [Big Blue Bus](http://gtfs.bigbluebus.com/current.zip).
+* We don't really use Partridge, but here's a link to their [repo](https://github.com/remix/partridge) in case you want to see where that handy diagram came from. Notice how trips are central to a GTFS feed.
+* Ignore references to `calitp_url_number`, `calitp_extracted_at`, or `calitp_deleted_at`. These refer to an older version of our warehouse. Learn about our current data warehouse [here](warehouse-starter-kit-page).
```
Comment (atvaccaro) [inline docs/warehouse/what_is_gtfs.md:14]: You could indicate the new columns these are replaced by

File: docs/warehouse/what_is_gtfs.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+* A "route" is a somewhat ambiguous concept! Transit providers have a lot of flexibility in branding their services. The same route can, and often does, have some trips following one path and some trips following another. GTFS has another concept of a "shape" which describes a path through physical space that one or more trips can follow.
+
+## Further GTFS Resources
+
+* [slides](https://docs.google.com/presentation/d/1fqIeXevb18T5s5k6XPxFbVEMHBPybeV29rFoFXROCw8/) from the video
```
Comment (atvaccaro) [inline docs/warehouse/what_is_gtfs.md:19]: nit: capitalize slides

**Review by atvaccaro — APPROVED** [review]
lgtm! I am being incredibly nitpicky

File: docs/warehouse/what_is_gtfs.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Lots of information in the warehouse comes from the General Transit Feed Specification, or GTFS.
+
+## Video Introduction
+
+### [Start by watching this amazing video that Laurie made!](https://www.youtube.com/watch?v=Us6j5GYoLtk)
+
+### Video Notes
+
+* Laurie mentions downloading an example feed to look at the files. This is a great idea! But when working with our data warehouse remember that you don't have to interact with raw GTFS feeds directly (lots of important work has been done for you!). Still, we reccomend taking a look at an example feed to understand what it looks like. Here's one from [Big Blue Bus](http://gtfs.bigbluebus.com/current.zip).
```
Comment (lauriemerrell) [inline docs/warehouse/what_is_gtfs.md:12]: I still see a typo warning here for `reccomend`

**Review by lauriemerrell — APPROVED** [review]
Thanks Eric!

**Comment by tiffanychu90** [comment]
* related to https://github.com/cal-itp/data-analyses/issues/708

**Pull request merged**